app/Modules/Newsletter/Model/NewsletterService.php

```diff
@@ -143,24 +136,25 @@ public function sendNewsletters(array $usersNewslettersIds)
 	{
 		$usersNewsletters = $this->userNewsletterModel->getAll()->wherePrimary($usersNewslettersIds)->fetchAll();
 		foreach ($usersNewsletters as $userNewsletter) {
-			$sendGrid = new SendGrid($this->apiKey);
-			$email = new Email;
 
-			$email->addTo($userNewsletter->user->email)
-				->setFrom($userNewsletter->from)
-				->setFromName('Eventigo.cz')
-				->setSubject($userNewsletter->subject)
-				->setCategory('newsletter')
-				->setHtml($userNewsletter->content);
-				//TODO: setText() - we should also send text format
+
+			$to = new Email(null, $userNewsletter->user->email);
+			$from = new Email('Eventigo.cz', $userNewsletter->from);
+
+			$subject = $userNewsletter->subject;
+			$content = new SendGrid\Content('text/plain', $userNewsletter->content); // @todo: or html?
```

Comment:
Ok, I've onyl upgraded sendgrid from 4 to 5 and never used it, so just asking to be sure :)